src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcViewDocumentClassifierPass.cs

```diff
@@ -31,12 +30,12 @@ public class MvcViewDocumentClassifierPass : DocumentClassifierPassBase
             {
                 // It's possible for a Razor document to not have a file path.
                 // Eg. When we try to generate code for an in memory document like default imports.
-                var checksum = BytesToString(codeDocument.Source.GetChecksum());
+                var checksum = Checksum.BytesToString(codeDocument.Source.GetChecksum());
                 @class.ClassName = $"AspNetCore_{checksum}";
             }
             else
             {
-                @class.ClassName = CSharpIdentifier.GetClassNameFromPath(filePath);
+                @class.ClassName = GetClassNameFromPath(filePath);
```

Comment:
Moved `CSharpIdentifier` in to Razor.Language, moved the cshtml path generation here